### ISSUE TITLE
Remove demos and libs directories from the repo

### ISSUE THREAD

**Issue by metteo** [vincenzopalazzo/material-ui-swing#140]
Both of those directories contain artifacts of the build. They should be published either to Maven Central or on release page here on GitHub.

I can create a PR with the requested change if it is approved.

**Comment by vincenzopalazzo** [comment]
Hi @metteo,

Yes, the library is on maven from one year, we developer uploaded the pre-release on this directory.
I haven't time to create a build with github action, but before to remove this I will search in the "issues doc" the links at this directory to remove it.

I hate the github repositories with dead links, In my opinion, this library is difficult to use at the moment because there isn't documentation and all people use this library like a normal L&F.
Before removing this, I fix the "issue doc" (all issue with the doc tag) with new reference and I have the intention to move this directory on the github action.

In conclusion, at the moment there is a few people that use this directory like referment because there was a method to upload the recent new jar on these directories

I have the intention to remove this but with more time, I don't want break people habits

**Comment by vincenzopalazzo** [comment]
I'm closing this issue because the branch main now have these changes.

Thanks.
